components/eam/src/control/iop_data_mod.F90

```diff
@@ -192,45 +166,35 @@ module scamMod
   logical*4, public ::  have_asdif    ! dataset contains asdif
   logical*4, public ::  scm_iop_srf_prop   ! use the specified surface properties
   logical*4, public ::  iop_dosubsidence ! compute Eulerian LS vertical advection
```

Comment:
Do we know if the subsidence option works?

Reply:
Yes, while there is no official unit test for this routine at this time, I have thoroughly tested/debugged this.  Many of our standard test cases (i.e. DYCOMS-RF01) rely on this routine.  Additionally, this routine really isn't new as the code was basically copied and pasted (with changed variable names) from the CAM3.5/CAM5 SCM Eulerian subsidence calculation from the old forecast.f90 module.